Core/DisconnectMeStore.swift

```diff
@@ -30,26 +30,26 @@ public class DisconnectMeStore {
 
     public init() {
         stringCache = ContentBlockerStringCache()
+        loadTrackers()
```

Comment:
Load the trackers in to memory on initialisation so that we can keep a reference to this.   The `networkNameAndCategory` function was moved in to this class and it made more sense and it uses the trackers property so that could get expensive when showing the trackers list in the UI.